src/scf-release/packages/suse-fuse/packaging

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -x
+
+DST=${BOSH_INSTALL_TARGET}
+SRC=/var/cache/zypp/packages
+
+# Clear the rpm cache. We want to see later only new files.
+zypper clean
+
+# Get fuse rpm and dependencies
+zypper --non-interactive --quiet install --download-only fuse
```

Comment:
Jenkins fails here with `16:59:57 compilation-suse-fuse > No provider of 'fuse' found.`
Have to check what set of repos are known here.
Difference to Vagrant which was ok.

Reply:
Should remove the `--quiet`

Reply:
Added diagnostics with commit 571a972ea60268849419855b642ed8be94ef47fa.
Currently suspect:

- Vagrant = opensuse stemcell, has predefined repos to look at
- Jenkins = sle stemcell, no repos, or other

Reply:
Confirmed. OpenSuse stemcell knows 3 repos, opensuse distribution + update, and stemcell repo.
SLE stemcell only knows the stemcell repo.

Reply:
By adding the relevant repos I I will also get the same fuse for both I guess. Using a fixed version should therefore be no problem.

Reply:
Ref commit c6471b01a4c78e864a42e30e27e0a898eba062b2, trialing

Reply:
Past the build stage now with commits 23cd7f909e4a886a1a3337403a91cabe999f79c6 and 8f344a3cb8b112b3bbaad9ebb76b03dc866a5a6e fixing a few zypper options.